frontend/shared/src/utils/dom.utils.ts

```diff
@@ -22,3 +19,24 @@ export const focusAndScroll = (elementId: string | undefined): void => {
     behavior: 'smooth',
   });
 };
+
+/**
+ * Focus and scroll to element using it's id
+ * @param elementId
+ * @returns void
+ */
+export const focusAndScroll = (elementId: string): void => {
+  // eslint-disable-next-line unicorn/prefer-query-selector
+  const element = document.getElementById(elementId);
+  return handleScroll(element);
+};
+
+/**
+ * Focus and scroll to any element using querySelector
+ * @param elementSelector
+ * @returns void
+ */
+export const focusAndScrollTo = (elementSelector: string): void => {
+  const element = document.querySelector(elementSelector);
+  return handleScroll(element as HTMLElement);
+};
```

Comment:
I don't really like the naming of the new function. Both of these scroll to an element, one by id and one by selector, but the only difference in the names is "to". Their names should be more distinct.